### PR TITLE
update/components/new-cta

### PR DESCRIPTION
**Pull request by worawutbat**
This PR adds a new CTA component.

### CODE REVIEW

File: public/components/ctas/8.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<aside class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center">
```
Comment (markmead) [inline public/components/ctas/8.html:1]: Classes that are not needed:

- `relative`
- `overflow-hidden`
- `lg:flex`
- `justify-center`

File: public/components/ctas/8.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="w-full p-12 text-center lg:w-2/3 sm:p-16 lg:p-24">
```
Comment (markmead) [inline public/components/ctas/8.html:2]: Not a fan of `w-full lg:w-2/3`. The child element uses `max-w-xl` so why are these needed?

File: public/components/ctas/8.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      <div class="max-w-xl mx-auto lg:ml-0">
```
Comment (markmead) [inline public/components/ctas/8.html:3]: Adding `lg:ml-0` here is causing you to need the `lg:flex justify-center` on the root element for this component.

Removing the previous classes highlighted and `lg:ml-0` solves that.

File: public/components/ctas/8.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+        <div class="p-2 border inline-block">community</div>
```
Comment (markmead) [inline public/components/ctas/8.html:5]: Not a fan of having content without any inner markup within a `<div>`.

Replace with a `<span>` or a `<strong>`, your choice.

File: public/components/ctas/8.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+        <p class="mt-4 text-2xl font-bold text-white sm:text-3xl w-2/3 text-center mx-auto">
```
Comment (markmead) [inline public/components/ctas/8.html:7]: Classes that are not needed:

- `w-2/3`
- `text-center`
- `mx-auto`

Should be a heading element, possibly a `<h2>`.

File: public/components/ctas/8.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+          Lorem, ipsum dolor sit amet consectetur.
+        </p>
+  
+        <p class="hidden lg:mt-4 lg:block mx-auto">
```
Comment (markmead) [inline public/components/ctas/8.html:11]: Don't need the `mx-auto`.

File: public/components/ctas/8.html [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Et, egestas
+          tempus tellus etiam sed. Quam a scelerisque amet ullamcorper eu enim et
+          fermentum,
+        </p>
+
+        <a
```
Comment (markmead) [inline public/components/ctas/8.html:17]: Could the `href=""` be replaced with `href="#"`?

Should `text-black` be `text-gray-900` so it pairs up nicely with the background color?

How does `hover:bg-yellow-400` look? And can we add `transition` to the classes for a smooth hover effect.

Comment (markmead) [reply public/components/ctas/8.html:17]: Probably want to consider `focus:` styling as well.

**Review by markmead — CHANGES_REQUESTED** [review]
Looks awesome, few things to clean up!

Apologies as most of the comments are things I need to fix on the other CTA components.

**Comment by markmead** [comment]
Pushed up a fix to the PR as it was using an old version of HyperUI.

**Comment by markmead** [comment]
Any update on this? Only a few small changes and then we can get this merged in! 🚀 @worawutbat

**Comment by worawutbat** [comment]
> Any update on this? Only a few small changes and then we can get this merged in! 🚀 @worawutbat

i fixed this PR by your suggestion, and already to merge and thank for your suggestion and review in this PR

**Comment by markmead** [comment]
Thanks @worawutbat! 🥳 

It looks like you've updated `public/components/ctas/8.html`, but the file should be `public/components/marketing-ctas/11.html`.

I've moved over your work and updated the component to match the new CTA markup, I just need you to apply those fixes to the new file and we're can ship it 🚢

File: public/components/ctas/8.html [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<aside
```
Comment (markmead) [inline public/components/ctas/8.html:1]: This file shouldn't be here.

File: public/components/marketing-ctas/11.html [hunk 9]
```diff
@@ -0,0 +1,28 @@
+<aside
```
Comment (markmead) [inline public/components/marketing-ctas/11.html:1]: Changes should be made to this file.

**Review by markmead — CHANGES_REQUESTED** [review]

File: public/components/marketing-ctas/11.html (hunk 9 above)
Comment (worawutbat) [reply public/components/marketing-ctas/11.html:1]: My bad, Thanks @markmead

File: public/components/ctas/8.html (hunk 1 above)
Comment (markmead) [reply public/components/ctas/8.html]: Oops, this hasn't been addressed @worawutbat

**Review by markmead — CHANGES_REQUESTED** [review]
Can't approve this while there's feedback waiting to be addressed.

Also, you've made everything a `<strong>` now which isn't correct.

**Review by markmead — CHANGES_REQUESTED** [review]
Why is there now an image? This isn't the original component you asked me to review.

File: public/components/marketing-ctas/11.html [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+  class="overflow-hidden bg-[url(/photos/tee-green-hanger-2.png)] bg-center bg-no-repeat bg-cover flex items-center justify-center"
+>
+  <div class="p-12 text-center max-w-xl sm:p-16 lg:p-24 text-gray-300">
+    <div class="max-w-xl mx-auto">
+      <div class="p-2 border inline-block">community</div>
```
[the remark this answers is not in the document]
Comment (worawutbat) [reply public/components/marketing-ctas/11.html]: > Why is there now an image? This isn't the original component you asked me to review.

sorry, i pushed PRs with wrong commit that trying to use new bg classname.

File: public/components/marketing-ctas/11.html [hunk 11]
```diff
@@ -0,0 +1,28 @@
+<aside
+  class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center"
+>
+  <strong class="p-12 text-center max-w-xl sm:p-16 lg:p-24 text-gray-300">
```
Comment (markmead) [inline public/components/marketing-ctas/11.html:4]: Not a `<strong>` tag.

File: public/components/marketing-ctas/11.html [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+    <strong class="max-w-xl mx-auto">
```
Comment (markmead) [inline public/components/marketing-ctas/11.html:5]: Not a `<strong>` tag.

**Review by markmead — CHANGES_REQUESTED** [review]
You're still not understanding what needs to be done.

Sorry, but I'll review this PR one more time, if there are still issues with the HTML then I will be closing it. I'd suggest you take a look at semantic HTML.

**Comment by worawutbat** [comment]
> You're still not understanding what needs to be done.
> 
> Sorry, but I'll review this PR one more time, if there are still issues with the HTML then I will be closing it. I'd suggest you take a look at semantic HTML.

Thanks for your suggestion, I fixed this PR by using the right semantic HTML.

File: public/components/marketing-ctas/11.html [hunk 13]
```diff
@@ -0,0 +1,28 @@
+<aside
+  class="relative justify-center overflow-hidden text-gray-300 bg-gray-900 lg:flex"
```
Comment (markmead) [inline public/components/marketing-ctas/11.html:2]: Unused classes have still not been removed.

File: public/components/marketing-ctas/11.html [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+>
+  <div class="max-w-xl p-12 text-center text-gray-300 sm:p-16 lg:p-24">
```
Comment (markmead) [inline public/components/marketing-ctas/11.html:4]: Don't need `max-w-xl` and `text-center` should be on the child class.

**Review by markmead — CHANGES_REQUESTED** [review]
Still, work to be done, I'm closing this PR as it's been through too many revisions. Please do finish it off based on **all** the feedback provided in this PR and submit again! 🎉

**Pull request closed without merging**